### PR TITLE
lz-analyze policy prior output

### PR DESCRIPTION
**Pull request by alreadydone**
Per request at https://github.com/gcp/leela-zero/pull/1772#discussion_r215905542
It's not my original work; the diff was communicated to me by 榜爷 (who operated LZ in Tencent competition).

### CODE REVIEW

File: src/UCTSearch.cpp [hunk 1]
```diff
@@ -67,6 +66,7 @@ class OutputAnalysisData {
     std::string m_move;
     int m_visits;
     int m_winrate;
+    std::string m_N_num; // policy prior
```
Comment (gcp) [inline src/UCTSearch.cpp]: Rename this to m_policy_prior please, and store it as a float. (Do the conversion at display time)

**Comment by gcp** [comment]
Note the point about storing it as a float and doing the conversion at display time, not when creating the storage struct.

(I prefer this because, for example, if you wanted to work with the struct and sort the output somehow, you'd need to redo the conversion twice)

**Comment by alreadydone** [comment]
Just made the change. Sorry somehow I neglected that part of your comment.

**Comment by gcp** [comment]
@alreadydone Thanks. Note that I changed the "N" in the output to "prior" as well, and changed the format to be consistent with "winrate".

**Pull request merged**